docs/book/src/developer/providers/v1.0-to-v1.1.md

```diff
@@ -36,7 +36,8 @@ are kept in sync with the versions used by `sigs.k8s.io/controller-runtime`.
 
 * Some controllers have been moved to internal to reduce their API surface. We now only
   surface what is necessary, e.g. the reconciler and the `SetupWithManager` func:
-    * [bootstrap/kubeadm/controllers](https://github.com/kubernetes-sigs/cluster-api/pull/5493) 
+    * [bootstrap/kubeadm/controllers](https://github.com/kubernetes-sigs/cluster-api/pull/5493)
```

Comment:
just to clarify are we intending to move all of them for 1.1?

Reply:
yes, I'm working to move everything in /controllers by this week/middle of the next week